metricbeat/metricbeat.reference.yml

```
@@ -516,6 +516,13 @@ metricbeat.modules:
   # By setting raw to true, all raw fields from the status metricset will be added to the event.
   #raw: false
 
+#-------------------------------- Nats Module --------------------------------
+- module: nats
+  metricsets: ["stats"]
+  period: 10s
+  hosts: ["localhost:8222"]
+  #stats_metrics_path: "/varz"
```

This should be as following as we prefix metricset specific configs normally like this:
```suggestion
  #stats.metrics_path: "/varz"
```